tests/conformance/conformance_test.go

```diff
@@ -32,6 +32,8 @@ import (
 	"github.com/containers/image/v5/transports"
 	"github.com/containers/image/v5/types"
 	"github.com/containers/storage"
+
```

Comment:
Not a blocker, but if we have to revise this PR for any reason, this blank line doesn't belong here.

Reply:
DOH, just saw this comment have inadvertently added it to a number of branches.  Will fix what I can without going nuts.